fio/func.sh

```
@@ -1,30 +1,39 @@
 }
 
-parse_lat() {
-        local OUTPUT_LAT=$1
-        RAND_READ_LAT=`cat $OUTPUT_LAT | jq '.jobs[0].read.lat_ns.mean'| cut -f1 -d.`
-        RAND_WRITE_LAT=`cat $OUTPUT_LAT | jq '.jobs[1].write.lat_ns.mean'| cut -f1 -d.`
-        SEQ_READ_LAT=`cat $OUTPUT_LAT | jq '.jobs[2].read.lat_ns.mean'| cut -f1 -d.`
-        SEQ_WRITE_LAT=`cat $OUTPUT_LAT | jq '.jobs[3].write.lat_ns.mean'| cut -f1 -d.`
-        CPU_IDLE_PCT_LAT=`cat $OUTPUT_LAT| jq '.cpu_idleness.system' | cut -f1 -d.`
+parse_read_bw() {
+	local OUTPUT_BW=$1
+	SEQ_READ_BW=`cat $OUTPUT_BW | jq '.jobs[0].read.bw_mean'| cut -f1 -d.`
```

Shellcheck

[warning] SEQ_READ_BW appears unused. Verify use (or export if used externally).

(SC2034)